src/test/java/io/outfoxx/swiftpoet/test/FileSpecTests.kt

```diff
@@ -244,10 +262,58 @@ class FileSpecTests {
       out.toString(),
       equalTo(
         """
-            class Test {
+          import Foundation
+          
+          class Test {
+
+            let a: Array
+            let b: Foundation.Array
+
+          }
+
+        """.trimIndent()
+      )
+    )
+  }
+
+  @Test
+  @DisplayName("Generates all required imports with conflicts")
+  fun testGeneratesAllRequiredImportsWithConflicts() {
+    val type =
+      TypeSpec.structBuilder("SomeType")
+        .addProperty(
+          PropertySpec.varBuilder(
+            "foundation_order",
+            typeName("Foundation.SortOrder")
+          ).build()
+        )
+        .addProperty(
+          PropertySpec.varBuilder(
+            "order",
+            typeName("some_other_module.SortOrder")
+          ).build()
+        )
+        .build()
+
+    val testFile = FileSpec.builder("Test", "Test")
+      .addImport("Foundation")
```

Comment:
nit: this should be removed as it might cause a false positive. This will prove that SwiftPoet does the right thing to emit `import Foundation`.

Reply:
👍